FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,44 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard self != nil else { return }
+
+			switch result {
+			case let .success((data, response)):
+				completion(FeedImageMapper.map(data, response))
+			case .failure:
+				completion(.failure(Error.connectivity))
+			}
+		}
+	}
+}
+
+private final class FeedImageMapper {
```

Comment:
`FeedImageMapper` instances has no specific behaviour. There's no value in initialising it but the code allows the developer to do so, `FeedImageMapper()` is absolutely valid code. 
In such cases when a type is used only as a namespace enums with no cases can be used to disallow instantiation at compile time so that `FeedImageMapper()` won't compile.

```suggestion
private enum FeedImageMapper {
```

Reply:
The suggested changes were made and pushed.

Reply:
Nice one!